dockers/docker-fpm-frr/frr/bgpd/bgpd.conf.j2

```diff
@@ -14,11 +14,15 @@
 !
 agentx
 !
+{% if DEVICE_METADATA.localhost.frr_mgmt_framework_config is defined and DEVICE_METADATA.localhost.frr_mgmt_framework_config == "true" %}
+{% include "bgpd.conf.db.j2" %}
```

Comment:
nit: can we have a comment line that indicates which template is being rendered? If

Reply:
Added comment lines on included template to identify the file being rendered